airflow/models/dagbag.py

```diff
@@ -455,5 +449,5 @@ def sync_to_db(self):
         """
         Save attributes about list of DAG to the DB.
         """
-        from airflow.models.dag import DAG
         DAG.bulk_sync_to_db(self.dags.values())
+        SerializedDagModel.bulk_sync_to_db(self.dags.values())
```

Comment:
We don't want to write Serialized DAGs in bulk !

Reply:
The implementation has not changed. Only the location of the method call has changed. Do you have another idea for the name?

Reply:
We already have SerializedDagModel.write_dag , can we not use that?

Reply:
I didn't want to add a loop to this method. In other places, we may also need to save many items with one call, e.g. tests. I will add that the previous code was also in the method that had the word "bulk" in the name. Bulk does not necessarily mean that this is done with a single database query. Bulk only means that the method accepts a list of elements.

Reply:
What do you think about sync_list_to_db?

Reply:
I am fine with `SerializedDagModel.bulk_sync_to_db` as long as you add some comments to say we don't intend to send bulk requests for serialized dags and the reason